README.md

```diff
@@ -355,6 +355,8 @@ the client used to send the request.
 | `client.insecure`                      | Whether to skip verifying the server's certificate chain and host name.     | `false`         |
 | `client.ignore-redirect`               | Whether to ignore redirects (true) or follow them (false, default).         | `false`         |
 | `client.timeout`                       | Duration before timing out.                                                 | `10s`           |
+| `client.client-certificate-file`       | Path to a client certificate (in PEM format) for mTLS configurations        | `""`            |
+| `client.client-private-key-file`       | Path to a client private key (in PEM format) for mTLS configurations        | `""`            |
```

Comment:
`client` is redundant in the name since the parent configuration name is already `client`.

Also, I'd like this to be under `tls` so that it matches the existing `web.tls.certificate-file` and `web.tls.private-key-file` configuration, so in other words, the configuration would look like `client.tls.certificate-file` and `client.tls.private-key-file`.

Reply:
I reworked this as requested and made a few other changes to the configs that align more with the other functions. Also added a few tests to verify the configuration options.

Reply:
very excited about this. @TwiN have you had time to review the updated PR? If all okay, could we expect a new release soon?